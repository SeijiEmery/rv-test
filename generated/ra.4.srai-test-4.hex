0000000    b3  90  20  00  93  d0  10  40  73  00  10  00                
0000014
