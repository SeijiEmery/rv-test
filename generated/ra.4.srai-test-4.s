sll 	x1, x1, x2
srai 	x1, x1, 1
ebreak
