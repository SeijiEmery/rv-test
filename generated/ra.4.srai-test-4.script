load /x 0 generated/ra.4.srai-test-4.hex
writereg 1 1
writereg 2 63
run 0 100000
readreg 1
readreg 2