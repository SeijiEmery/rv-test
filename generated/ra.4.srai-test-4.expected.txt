R1 = 13835058055282163712
R2 = 63
